Jenkinsfile

```diff
@@ -209,9 +209,9 @@ pipeline {
                         script {
                             // If this build is not a pull request, run sonar scan. otherwise run incremental scan
                             if (env.CHANGE_ID == null) {
-                                sh 'mvn -q -B -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=ddf -Dsonar.exclusions=${COVERAGE_EXCLUSIONS} -pl !$DOCS,!$ITESTS $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```

Comment:
What are the consequences of removing `org.jacoco:jacoco-maven-plugin:prepare-agent`?

Reply:
Good question. I'll research a little more. It looks like it was putting out our extra arguments: https://www.eclemma.org/jacoco/trunk/doc/prepare-agent-mojo.html